RecoPixelVertexing/PixelTrackFitting/plugins/PixelNtupletsFitterProducer.cc

```diff
@@ -11,43 +11,41 @@
 #include "FWCore/ParameterSet/interface/ParameterSetDescription.h"
 
 #include "RecoPixelVertexing/PixelTrackFitting/interface/PixelFitter.h"
-#include "RecoPixelVertexing/PixelTrackFitting/interface/PixelFitterByRiemannParaboloid.h"
+#include "RecoPixelVertexing/PixelTrackFitting/interface/PixelNtupletsFitter.h"
 
 #include "MagneticField/Engine/interface/MagneticField.h"
 #include "MagneticField/Records/interface/IdealMagneticFieldRecord.h"
+#include "RecoTracker/TkMSParametrization/interface/PixelRecoUtilities.h"
 
-class PixelFitterByRiemannParaboloidProducer: public edm::global::EDProducer<> {
+class PixelNtupletsFitterProducer: public edm::global::EDProducer<> {
 public:
-  explicit PixelFitterByRiemannParaboloidProducer(const edm::ParameterSet& iConfig)
-    : useErrors_(iConfig.getParameter<bool>("useErrors")),
-    useMultipleScattering_(iConfig.getParameter<bool>("useMultipleScattering"))
+  explicit PixelNtupletsFitterProducer(const edm::ParameterSet& iConfig)
+    : useRiemannFit_(iConfig.getParameter<bool>("useRiemannFit"))
   {
     produces<PixelFitter>();
   }
-  ~PixelFitterByRiemannParaboloidProducer() override {}
+  ~PixelNtupletsFitterProducer() override {}
 
   static void fillDescriptions(edm::ConfigurationDescriptions& descriptions) {
     edm::ParameterSetDescription desc;
-    desc.add<bool>("useErrors", true);
-    desc.add<bool>("useMultipleScattering", true);
-    descriptions.add("pixelFitterByRiemannParaboloidDefault", desc);
+    desc.add<bool>("useRiemannFit", false);
```

Comment:
The configuration would be more self-documenting if we'd use a string here, but that can be done later as well.

Reply:
yes sure, as you suggested in #265

Reply:
I had more in mind something along `desc.add<std::string>("fitter", "BrokenLine");` with valid values being e.g. (for now) "BrokenLine", "RiemannFit", but it's not really critical (and may be a matter of taste).